x/confidentialtransfers/types/params.go

```diff
@@ -9,8 +9,8 @@ import (
 // DefaultEnableCtModule is the default value for the EnableCtModule flag.
 const DefaultEnableCtModule = true
 
-// DefaultRangeProofGasMultiplier is the default value for RangeProofGasMultiplier param.
-const DefaultRangeProofGasMultiplier = uint32(10)
+// DefaultRangeProofGasCost is the default value for RangeProofGasCost param.
+const DefaultRangeProofGasCost = uint32(1000000)
```

Comment:
Nit: we could use uint64 instead (to use the same type as Gas amount)

Reply:
Good call, made this change